src/modules/ui/core/tooltip/tooltip-container.ts

```diff
@@ -3,18 +3,17 @@ import htmlToElement from '../htmlToElement';
 import Tooltip from './tooltip';
 import Stylable from '../stylable';
 
-import {
-  ITooltipPosition,
-  ITooltipPositionFunction,
-  ITooltipPositionPlacement,
-} from './types';
-
 import styles from './tooltip-container.scss';
 
+enum TooltipPositionPlacement {
```

Comment:
duplicated in `src/modules/ui/core/tooltip/types.d.ts`

Reply:
@bodia-uz it's not